core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -1289,7 +1290,11 @@ public void resumeWorkflow(String workflowId) {
                     "Current status is " + workflow.getStatus().name());
         }
         workflow.setStatus(WorkflowStatus.RUNNING);
+        workflow.setLastRetriedTime(System.currentTimeMillis());
```

Comment:
What is the significance of this change? Does it affect the workflow timeout in any way?

Reply:
yes, this ensures that the workflow will not be timed out because of the time it was PAUSED. Currently, we do not have an execution time measurement, this acts as a stop-gap and is used similarly in other uses.